recipes/qtmolecularnetwork/meta.yaml

```diff
@@ -0,0 +1,117 @@
+{% set version = "0.6.2" %}
+{% set rdkit_version = "2022.03" %}
+
+package:
+    name: qtmolecularnetwork
+    version: {{ version }}
+
+source:
+    url: https://github.com/metgem/QtMolecularNetwork/archive/v{{ version }}.tar.gz
+    sha256: 947557699055d58d41c83903069e6cfd91d8b3a7db6634e3459022d4fb3f06e0
+    patches:
+        - 0001-libcxx.patch                   # [linux]
+
+build:
+    number: 0
+    run_exports:
+    # FIXME: Adjust pin according to future ABI compatability guarantees
+      - {{ pin_subpackage('qtmolecularnetwork', max_pin='x.x.x') }}
+    ignore_run_exports_from:
+    # Just used to satisfy CMake; not used in shared library
+      - cairo
+      - freetype
+
+requirements:
+    build:
+        - cmake
+        - {{ compiler('cxx') }}
+        - make                                # [not win]
+        - {{ cdt('mesa-libgl-devel') }}       # [linux]
+        - {{ cdt('mesa-dri-drivers') }}       # [linux]
+        - {{ cdt('libselinux') }}             # [linux]
+        - {{ cdt('libxdamage') }}             # [linux]
+        - {{ cdt('libxxf86vm') }}             # [linux]
+        - {{ cdt('libxext') }}                # [linux]
+    host:
+        - cairo
+        - freetype
+        - boost-cpp
+        - eigen
+        - qt-main >=5.15
+        - rdkit
+        - rdkit-dev
+    run:
+        - {{ pin_compatible('rdkit', max_pin='x.x.x') }}
```

Comment:
```suggestion
        - rdkit
        - rdkit-dev  # [win]
    run:
        - {{ pin_compatible('rdkit', max_pin='x.x') }}
        - {{ pin_compatible('rdkit-dev', max_pin='x.x') }}  # [win]
```

Reply:
I don't know what is causing the conflict when pinning to the same version, but relaxing to minor version as you did earlier seems to allow enough flexibility.

Reply:
@carterbox pinning does not work and I can't manage to find why. Do you have any idea?